src/components/Form/Inputs/Date.js

```diff
@@ -1,337 +1,8 @@
 import React from "react";
-import styled, { css } from "styled-components";
-import { getDate, getDay, isSameMonth, isToday, format, parseISO } from "date-fns-2";
-import { FormattedDate, injectIntl } from "react-intl";
-import Kalendaryo from "kalendaryo";
-import { getThemeProp, ifFlag, switchEnum, memoize } from "../../../utils";
-import withClickOutside from "../../../hocs/withClickOutside";
-import useToggle from "../../../hooks/useToggle";
-import Icon from "../../Icon";
-import { FormInput, getEventUpdater } from "./Text";
-import { ButtonWrapper, InputButton } from "./FieldButtons";
+import DatePicker from "../../MaterialUI/Inputs/DatePicker";
 
-// TODO: Calendar dialog on focus
+// Component is replaced with DatePicker
```

Comment:
Revert Date file as it was originally please.

Reply:
Spoke to Alain...  he will eventually change DateInput to DatePicker, but would like to have this "stub" component for now.  The original DateInput was using an outdated component called kalendaryo.